src/components/Venue.js

```diff
@@ -32,7 +32,7 @@ class Venue extends Component {
       restaurantSlug: props.slug,
       restaurantId: props.post_id,
       restaurantTerms: '',
-      restaurantLink: 'http://nationalburgerday.co.uk/' + window.__INITIAL_STATE__.app.map.slug + '/' + props.slug,
+      restaurantLink: 'https://nationalburgerday.co.uk/' + window.__INITIAL_STATE__.app.map.slug + '/' + props.slug,
```

Comment:
Do you need to be explicit here of protocol? could you match protocol so `window.location.protocol` or just make the links relative?